.vscode/settings.json

```diff
@@ -21,6 +21,7 @@
   "prettier.bracketSpacing": false,
   "prettier.singleQuote": true,
   "prettier.printWidth": 80,
+  "prettier.trailingComma": "all",
```

Comment:
Thank you for the clarification. I thought that we support plain JS files for Node 6.x, which I ran into trailing comma issues for function args. 

Feel free to merge then.